apps/fluent-tester/src/FluentTester/TestComponents/Avatar/BasicAvatar.tsx

```diff
@@ -9,12 +9,14 @@ import {
   AvatarActiveAppearance,
 } from '@fluentui-react-native/experimental-avatar';
 import { PresenceBadgeStatuses, PresenceBadgeStatus } from '@fluentui-react-native/badge';
-import { Switch, View, Text, Picker, ColorValue, Platform } from 'react-native';
+import { Switch, View, Text, ColorValue, Platform } from 'react-native';
+import { Picker } from '@react-native-picker/picker';
 import { satyaPhotoUrl, undefinedText } from './../PersonaCoin/styles';
 import { commonTestStyles as commonStyles } from '../Common/styles';
 import { useTheme } from '@fluentui-react-native/theme-types';
 import TestSvg from '../../test-data/test.svg';
 import { SvgIconProps } from '@fluentui-react-native/icon';
+import { MenuPicker } from '../Common/MenuPicker';
```

Comment:
(Feedback is for after MenuPicker / StyledPicker are moved to a separate file / component)

When metro does it's bundling, It will see a dependency on "@react-native-picker/picker" for all platforms, and include that in the final bundle. It's not smart enough to know you did a Platform.OS check for win32 on line 63. That won't cause an error, but it will be unnecessary bloat to the final bundle size (why bundle the JS code for Picker if it's never used?).
What you _can_ do is make platform specific files, and only. have the import for the platforms that need it. So you could have your `Picker.tsx` import react-native-picker/picker and render `StyledPicker`, while `Picker.win32.tsx / Picker.macos.tsx` import MenuButton, and render `MenuPicker`